man/man1/kbsecret-cp.1.ronnpp

```diff
@@ -0,0 +1,26 @@
+kbsecret-cp(1) - copy kbsecret(1) record(s)
```

Comment:
You can remote the parens around `(s)`, for consistency with `kbsecret list` and others.

Reply:
This is the last thing that needs to be addressed, and then we should be good to go 👍

Reply:
Updated, let me know if there is anything else that needs to be fixed.